src/main/java/bookstore/dto/BookDto.java

```diff
@@ -0,0 +1,7 @@
+package bookstore.dto;
+
+import java.math.BigDecimal;
+
+public record BookDto(Long id, String title, String author, String isbn,
```

Comment:
a general recommendation on record formatting

if your record has 1-2 fields, you can:
```
public record SomeRecord(String field1, Integer field2) {
```
if your record has more than 2 fields:
```
public record SomeRecord(String field1, 
                         int field2,
                         Double field3,
                         boolean field4) {

```
or alrternative:
```
public record SomeRecord(
         String field1, 
         int field2,
         Double field3,
         boolean field4) {
```